api/useradminservice.yaml

```diff
@@ -711,6 +720,24 @@ components:
         absent:
           type: boolean
 
+    Sort:
+      type: object
+      properties:
+        field:
+          type: string
+          example: 'firstName|lastName|username|email'
```

Comment:
lowercase or uppercase? https://happylife.develop.onlineberatung.net/service/useradmin/consultants/?page=1&perPage=10&order=ASC&field=firstName -> 400 error

Reply:
my current frontend version sends uppercase and it works

Reply:
I integrated ignorecase for the validation but you can leave it uppercase i think ;-)